app/views/text_components/_form.html.haml

```diff
@@ -49,6 +49,9 @@
                 = f.input :from_day
               .col
                 = f.input :to_day
+            .row
+              .col
```

Comment:
The `.row` and `.col` wrapping divs are completely unnecessary in this case. You only need them if you want to create a grid with two or more columns like in the example above. In this case they only add bloat and make the markup harder to read.